### PR TITLE
Nutrition compartments

### PR DESCRIPTION
**Pull request by FrankApiyo**
This PR adds the compartment sub-module of the nutrition module.

### CODE REVIEW

File: src/components/DataCircleCard/test/__snapshots__/index.test.tsx.snap [hunk 1]
```diff
@@ -28,13 +28,14 @@ exports[`DataCircleCard must render correctly 1`] = `
         >
           <li
             className="red"
+            key="0"
           >
             <Link
               onClick={[Function]}
-              to="/pregnancy_compartments/hierarchicaldata/Pregnancy/high/test title/1/down/eccfe905-0e03-4188-98bc-22f141cccd0e/1"
+              to="/pregnancy_compartments/hierarchicaldata/Pregnancy/high/test title/undefined/down/undefined/undefined"
```
Comment (cKellyDesign) [inline src/components/DataCircleCard/test/__snapshots__/index.test.tsx.snap]: Seems like something is breaking in the tests

File: src/components/ReportTable/index.tsx [hunk 2]
```diff
@@ -244,14 +264,30 @@ class ReportTable extends Component<Props, State> {
         <Row id={'chart'}>
           <WeightAndHeightChart
             weights={
-              this.getWeightsArray(this.state.pregnancyEventsArray)[this.state.currentPregnancy]
+              this.getWeightsArray(this.state.pregnancyEventsArray, 'weight')[
```
Comment (cKellyDesign) [inline src/components/ReportTable/index.tsx]: Looks like `'weight'` is a magic string here

File: src/components/ReportTable/index.tsx [hunk 3]
```diff
@@ -244,14 +264,30 @@ class ReportTable extends Component<Props, State> {
...
+            chartWrapperId={this.props.isNutrition ? 'nutrition-chart' : 'pregnancy-chart'}
+            title={this.props.isNutrition ? CHILD_WEIGHT_MONITORING : MOTHER_WEIGHT_TRACKING}
+            legendString={this.props.isNutrition ? CHILD_WEIGHT : MOTHERS_WEIGHT}
             units={KG}
             xAxisLabel={WEIGHT}
           />
+          {this.props.isNutrition ? (
+            <WeightAndHeightChart
+              weights={
+                this.getWeightsArray(this.state.pregnancyEventsArray, 'height')[
```
Comment (cKellyDesign) [inline src/components/ReportTable/index.tsx]: Magic string

Comment (FrankApiyo) [reply src/components/ReportTable/index.tsx]: no more magic :blush:

File: src/components/DataCircleCard/test/__snapshots__/index.test.tsx.snap (hunk 1 above)
Comment (FrankApiyo) [reply src/components/DataCircleCard/test/__snapshots__/index.test.tsx.snap]: fixed!

File: src/containers/LogFace/index.tsx [hunk 4]
```diff
@@ -293,7 +294,7 @@ export class LogFace extends React.Component<PropsInterface, State> {
     return (
       <div className="logface-content">
         <div>
-          <h2 id="logface_title">{`Log Face - ${this.props.header}`}</h2>
+          <h2 id="logface_title">{`${LOG_FACE} - ${this.props.module}`}</h2>
```
Comment (cKellyDesign) [inline src/containers/LogFace/index.tsx]: @FrankApiyo looks like this broke the snapshot below

File: src/containers/LogFace/tests/__snapshots__/index.test.tsx.snap [hunk 5]
```diff
@@ -665,7 +665,7 @@ exports[`components/ConnectedHeader renders correctly: logface title 1`] = `
 <h2
   id="logface_title"
 >
-  Log Face - Pregnancy
+  Log Face -
```
Comment (cKellyDesign) [inline src/containers/LogFace/tests/__snapshots__/index.test.tsx.snap]: Looks like something is undefined?

File: src/containers/LogFace/index.tsx (hunk 4 above)
Comment (FrankApiyo) [reply src/containers/LogFace/index.tsx]: @cKellyDesign  I didn't pass the correct prop in a test. now fixed!

**Comment by moshthepitt** [comment]
@FrankApiyo why are we still maintaining the `miecd_vietnam` branch?  Can we not merge it now?

**Pull request merged**